index.html

```diff
@@ -3,10 +3,11 @@
 <head>
  
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
- <title>JS</title>
+<title>JS</title>
 <meta name="keywords" content="" />
 <link href="css/style.css" type="text/css" rel="stylesheet"  media="screen" />
- 
+<script type="text/javascript" src="js/js.js"></script>

```

Comment:
1) You includes this fils twice. One of them should be remove.
2) You should put js scripts on the bottom of page. I should load after dom loaded. It's best practice.